docs/rules/rego.md

```diff
@@ -0,0 +1,7 @@
+Create a `.rego` file in the rules directory that has the following Rego Rules (in this context rules are Rego's language constructs):
+
+1. `__rego_metadoc__`: A *document* rule that defines the rule's metadata (based on [WIP Rego convention](https://hackmd.io/@ZtQnh19kS26YiNlJLqKJnw/H1gAv5nB) for describing policy metadata).
```

Comment:
the link returns not found

Reply:
you can remove whatever's in parenthesis